packages/config/config/devices/0x0063/14322.json

```diff
@@ -0,0 +1,139 @@
+// GE/Jasco 14322
+// Jasco In-Wall Smart Dimmer (White Toggle)
+{
+	"manufacturer": "GE/Jasco",
+	"manufacturerId": "0x0063",
+	"label": "14322",
+	"description": "Jasco In-Wall Smart Dimmer (White Toggle)",
+	"devices": [
+		{
+			"productType": "0x4944",
+			"productId": "0x3137",
+			"zwaveAllianceId": [1372, 2123]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Local Load",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Double Tap",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"4": {
+			"label": "Invert Switch",
+			"description": "Invert the default On/Off rocker settings.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal Operation",
+					"value": 0
+				},
+				{
+					"label": "Invert",
+					"value": 1
+				}
+			]
+		},
+		"7": {
+			"label": "Dim Rate Steps (Z-Wave)",
+			"description": "Number of steps or levels",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Dim Rate Timing (Z-Wave)",
+			"description": "Timing of steps",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 255,
+			"defaultValue": 3,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Dime Rate Steps (Manual)",
+			"description": "Number of steps or levels",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Dim Rate Timing (Manual)",
+			"description": "Timing of steps",
+			"valueSize": 2,
```

Comment:
Is this really a size 2, but only a maxValue of 255? Seems weird.

Reply:
This might affect the other variants of this parameter too.

Reply:
That's what the printed manual says for all models, for all dim rate adjustment:

Parameter 8: Dim Rate Adjustments
Dim Rate Adjustments when receiving a Z-Wave dim command
Size: 2 Byte, Default Value: 3
Setting | Description
-- | --
1 - 255 | timing of steps

One version of the manual adds this note:

PLEASE NOTE: Values should be entered using their signed values only, which means for example that a configuration value of 1 byte in size must have values in the range -128 to 127. Values specified here above or below the range specified by the value's size need to be converted to their 2's compliment value, or if your controller supports it, may be entered using hexadecimal notation.